### PR TITLE
Adding test blocks into the playground

### LINKED ISSUE #1626: Test blocks in Playground
### Problem statement

There are several block configurations that never get regularly tested because they are not present in the standard blocks. Examples:

 * Empty block with a mutator (#1608).
 * Very long list drop down.
 * Drop down with images.
 * Date field blocks.
 * Angle field blocks.

### Proposed Solution

Add a new option "Test Blocks" to the playground toolbox-type selection. Selecting it will restart the workspace with categories of various block configurations that need testing. This keeps these block accessible testing, without modifying the standard block categories.

These blocks and their toolbox should be stored in a manner editable by the new devtools.

### PR DESCRIPTION
**Pull request by AnmAtAnm**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

#1626

Adds a number of test blocks to the playground for testing block configurations (and edge cases) not covered by the standard blocks.

### Proposed Changes

Adds a new toolbox option to the playground, which includes only test blocks.
Does not change the blocks shown in other toolbox configurations.

Blocks include:
 * Empty block (with and without a mutator)
 * Various configurations of number blocks.
 * Long drop-down fields and image drop-down fields.
 * Angle and date fields.
 * Image fields, including networked images and mis-specified images.

### Reason for Changes

Provides an easy way to test these cases, encouraging more regular spot testing.

### Test Coverage

Test blocks in this PR reference some network images that are not yet online. However, I tested them with URLs from the android project before setting them up with blockly-demo URL.

Tested on:
 * Desktop Chrome

### CODE REVIEW

File: tests/blocks/test_blocks.js [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+  {
+    "type": "test_with_lots_of_network_icons",
+    "message0": "Lots of network icons: %1 %2 %3 %4 %5 %6 %7 %8 %9 %10 %11 %12 %13 %14 %15 %16 %17 %18",
+    "args0": [
+      {
+        "type": "input_dummy"
+      },
+      {
+        "type": "field_image",
+        "src": "https://blockly-demo.appspot.com/static/media/test_a.png",
```
Comment (picklesrus) [inline tests/blocks/test_blocks.js]: can these use the media url thingy?

**Comment by AnmAtAnm** [comment]
I made the letter images. The LED icon I pulled as creative commons.

Imagess on Android repo:
 * https://github.com/google/blockly-android/tree/gh-pages/
 * https://github.com/google/blockly-android/tree/develop/blocklydemo/src/main/assets/images

Block definitions:
 * https://github.com/google/blockly-android/blob/develop/blocklydemo/src/main/assets/sample_sections/mock_block_definitions.json
 * https://github.com/google/blockly-android/blob/develop/blocklylib-core/src/main/assets/default/test_blocks.json

**Comment by picklesrus** [comment]
LGTM

**Comment by AnmAtAnm** [comment]
Travis failure unrelated to code:

```
{ Error: Couldn't connect to selenium server
    at getHTML("#closureTestRunnerLog") - test_runner.js:17:2
  details: undefined,
  message: 'connect ECONNREFUSED 127.0.0.1:4444',
  type: 'RuntimeError',
  seleniumStack: 
   { status: -1,
     type: 'ECONNREFUSED',
     orgStatusMessage: 'connect ECONNREFUSED 127.0.0.1:4444',
     message: 'Couldn\'t connect to selenium server' } }
```
